.circleci/config.yml

```diff
@@ -20,11 +20,12 @@ jobs:
       - run: npm test
   api_tests:
     docker:
-      - image: circleci/golang:1.13
+      - image: golangci/golangci-lint:v1.23.8
```

Comment:
Happy to install this another way. There's a few suggestions here:

https://github.com/golangci/golangci-lint#binary